olm.sh

```diff
@@ -9,3 +9,7 @@ operator-sdk generate bundle \
     --output-dir bundles/redhat-marketplace/"$RELEASE" \
     --channels stable \
     --overwrite
+
+# Add needed annotations for redhat marketplace
+yq -i e '.metadata.annotations."marketplace.openshift.io/remote-workflow" |= "https://marketplace.redhat.com/en-us/operators/minio-directpv-operator-rhmp/pricing?utm_source=openshift_console"' bundles/redhat-marketplace/"$RELEASE"/manifests/minio-directpv.clusterserviceversion.yaml
+yq -i e '.metadata.annotations."marketplace.openshift.io/support-workflow" |= "https://marketplace.redhat.com/en-us/operators/minio-directpv-operator-rhmp/support?utm_source=openshift_console"' bundles/redhat-marketplace/"$RELEASE"/manifests/minio-directpv.clusterserviceversion.yaml
```

Comment:
If `e` is a sub command, is it possible to use expanded name instead of abbreviated name?